test/functional/editions_controller_test.rb

```diff
@@ -64,4 +64,55 @@ class EditionsControllerTest < ActionController::TestCase
       assert_equal EditionsController.new.method(:metadata).super_method.name, :show
     end
   end
+
+  context "when 'restrict_access_by_org' feature toggle is disabled" do
+    setup do
+      test_strategy = Flipflop::FeatureSet.current.test!
```

Comment:
We can move the test_strategy to the top most setup block of the class, avoids repeating.

Reply:
This context block is explicitly for when the feature toggle is disabled, so I think it makes sense for the test_strategy setup to be here. Technically it shouldn't be needed since the toggle will be off by default, but it felt a bit weird to have the context say the feature toggle is off and then not actually do anything to turn it off.